server/repository/src/db_diesel/requisition/requisition_row.rs

```diff
@@ -35,13 +35,20 @@ table! {
         min_months_of_stock -> Double,
         approval_status -> Nullable<crate::db_diesel::requisition::requisition_row::RequisitionRowApprovalStatusMapping>,
         linked_requisition_id -> Nullable<Text>,
-        is_sync_update -> Bool,
         program_id -> Nullable<Text>,
         period_id -> Nullable<Text>,
         order_type -> Nullable<Text>,
     }
 }
 
+table! {
+    #[sql_name = "requisition"]
+    requistion_is_sync_update (id) {
```

Comment:
@andreievg oh. just notice the spelling on this. The requisition is missing an 👁️